build/nuke/Extensions/SolutionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Linq;
+using Nuke.Common.ProjectModel;
+
+namespace Extensions
+{
+    public static class SolutionExtensions
+    {
+        public static Project[] GetIntegrationTests(this Solution solution)
+        {
+            return solution
+                .GetProjects("IntegrationTests.*")
+                // Skip helpers project
+                .Where(x => !x.Name.Equals(Projects.Tests.IntegrationTestsHelpers, StringComparison.OrdinalIgnoreCase))
+                // Skip .NET Fx integrations
+                .Where(x => !x.Path.ToString().Contains("aspnet", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should the name we're checking here be more specific? It seems like this could accidentally exclude aspnet core tests.

Reply:
@nrcventura true, what do you think of checking directory path instead of project path? 
Changing `x => x.Path` to `x => x.Directory`. It's not going to eliminate the problem, but should reduce naming conflicts greatly - as only folders match.

Reply:
Would it be just as easy to use the TargetFrameworks [Project extension method](https://nuke.build/api/Nuke.Common/Nuke.Common.ProjectModel.ProjectExtensions.html#Nuke_Common_ProjectModel_ProjectExtensions_GetTargetFrameworks_Nuke_Common_ProjectModel_Project_) to perform the check? I think that would more closely match your intention.

Reply:
@nrcventura thanks! I have been looking for this functionality for a while. That's of curse much better.

Reply:
UPD: @nrcventura unfortunately TargetFramework doesn't tell us if it's ASP.NET FX or .NET FX Console. Currently all except StartupHook is including .net461. If you have any better ideas, we can try these but otherwise I would go with `x.Directory` option and come back later to figure out how to group asp.net fx, .net fx console and others

Reply:
I was thinking that we might be able to do something like (in pseudocode) `.Where(x => !x.TargetFrameworks.All(f => f.ContainsSubstring("net4"))`, so that we are excluding projects where all of the target frameworks are .net framework (as opposed to .net core, .net standard, or .net 5+).

Reply:
Another solution could be to rename the folder to `aspnetfx`, so we can distinguish between that and `aspnetcore`

Reply:
`project.GetTargetFrameworks()` actually gets me idea, that it's based on `project.GetProperty()` probably. which means I might be able to do something like `.Where(x => !x.GetProperty<bool>("IsAspNetIntegration"))`

Reply:
If you have some way to define the `IsAspNetIntegration` property without needing to remember to update each project file, that could work. I'd recommend checking for a property that is built-in or relying on our folder/naming structure so it will be hard to mess up

Reply:
The plan is to add props file to aspnet folder, so all projects auto gain that prop. Have to test it first, in theory it works, but if in practice `project.GetProperty()` is pure csproj xml parsing then I guess it wont work. The same thing is actually with TargetFrameworks as these are inherited also from some lower level .props and not explicitly defined in csproj.

Reply:
If we are able to use the prop approach, should the prop be more generic than `IsAspNetIntegration` and follow the `WindowsOnly` approach used elsewhere?

Reply:
Added `WindowsOnly` prop to `Directory.Build.props`, seems working fine.